tensorflow/c/c_api.cc

```diff
@@ -1667,6 +1667,25 @@ void TF_DeleteGraph(TF_Graph* g) {
   if (del) delete g;
 }
 
+size_t TF_GraphOperationsCount(TF_Graph* graph) {
```

Comment:
I've newly arrived to TF, so i'm not sure i totally grok the graph representation. Looking at the implementation of `TF_GraphNextOperation`, it is both (potentially) returning the next operation, and also (potentially) altering the position value by greater than +1 per invocation (for what conditions, i won't understand until i understand the graph representation.)

Given that, the suggested iteration implementation where we rely on a JNI implementation that receives a position argument that we continually increment at the Java layer, seems like it might fall afoul.

That being written, i'd obviously cede to you if you told me that this usage of position won't fail us.